easybuild/easyconfigs/i/IDBA-UD/IDBA-UD-1.1.3-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,48 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+# "make install" doesnt copy all the compiled binaries so we use the "MakeCp" easyblock
+# to be sure everything is copied and we run ./configure in prebuildopts
+
+# modified by Tom Strempel
+easyblock = 'MakeCp'
+
+name = 'IDBA-UD'
+version = '1.1.3'
+
+homepage = 'https://i.cs.hku.hk/~alse/hkubrg/projects/idba_ud/'
+description = """ IDBA-UD is a iterative De Bruijn Graph De Novo Assembler for Short Reads
+ Sequencing data with Highly Uneven Sequencing Depth. It is an extension of IDBA algorithm.
+ IDBA-UD also iterates from small k to a large k. In each iteration, short and low-depth
+ contigs are removed iteratively with cutoff threshold from low to high to reduce the errors
+ in low-depth and high-depth regions. Paired-end reads are aligned to contigs and assembled
+ locally to generate some missing k-mers in low-depth regions. With these technologies, IDBA-UD
+ can iterate k value of de Bruijn graph to a very large value with less gaps and less branches
+ to form long contigs in both low-depth and high-depth regions."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
```

Comment:
@tstrempel can you move this to `GCC` toolchain, so we can have an `iccifort` version as well?

Reply:
@smoors Yes, I'll do it.
For Future Reference: When should I use `GCC` and when should I use `GCCcore`?

Reply:
@tstrempel generally: for software that is not performance critical, GCCcore is preferred so that it can be reused, as much as possible.
if it is performance critical, use GCC so performance gains may be possible with other compilers. I suspect this is the case for IDBA-UD?